### PR TITLE
Move from webpack to esbuild as a demonstration of the possibility

### PR DESCRIPTION
**Pull request by gausie**
Webpack: 3.00 seconds to make a 826KB build
Esbuild: 1.76 seconds to make a 705KB build

This also includes some bundle-size fixed I made because of bad imports from lodash applied to both cases

### CODE REVIEW

**Comment by horrible-little-slime** [comment]
I don't think I can meaningfully review this, but I will say, I care more about build size than build time. It's nice to have quick builds when you're testing things, but it doesn't really matter that much at the end of the day. Sizes also don't matter, but at least that's something that is in a constant state of mattering slightly.

File: src/resources/2020/Guzzlr.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Item, Location, mallPrice, runChoice, use, visitUrl } from "kolmafia";
-import { maxBy } from "lodash";
+import maxBy from "lodash/maxBy";
```
Comment (horrible-little-slime) [inline src/resources/2020/Guzzlr.ts:2]: Is this general spring cleaning, or is this something that we'll specifically want to start doing once we switch to esbuild?

Comment (gausie) [reply src/resources/2020/Guzzlr.ts:2]: We should be doing this now

Comment (gausie) [reply src/resources/2020/Guzzlr.ts:2]: Otherwise we massively inflate the build size

**Comment by horrible-little-slime** [comment]
#414 414

**Pull request closed without merging**